tests/routes/test_place_reviews.py

```diff
@@ -91,8 +94,11 @@ def test_post_place_review_with_wrong_payload(client):
     assert "value is not a valid enumeration member" in response.json[0]["msg"]
 
 
-def test_delete_place_review(runner):
-
+@mock.patch("lightsoff_api.commands.app.app_context")
```

Comment:
We should mock it with existing app context provided by the db_fixture but didn't managed to succeed